Azure.ResourceManager.Core/Resources/ResourceType.cs

```diff
@@ -48,68 +61,130 @@ public ResourceType Parent
+        /// Implicit operator for initializing a <see cref="ResourceType"/> instance from a string.
+        /// </summary>
+        /// <param name="other"> String to be conferted into a <see cref="ResourceType"/> object. </param>
         public static implicit operator ResourceType(string other)
         {
             return new ResourceType(other);
         }
 
+        /// <summary>
+        /// Compares a <see cref="ResourceType"/> object with a <see cref="string"/>.
+        /// </summary>
+        /// <param name="source"> <see cref="ResourceType"/> object. </param>
+        /// <param name="target"> String. </param>
+        /// <returns> True if they are equal, otherwise False. </returns>
         public static bool operator ==(ResourceType source, string target)
         {
             return source.Equals(target);
         }
 
+        /// <summary>
+        /// Compares a <see cref="string"/> with a <see cref="ResourceType"/> object.
+        /// </summary>
+        /// <param name="source"> String. </param>
+        /// <param name="target"> <see cref="ResourceType"/> object. </param>
+        /// <returns> True if they are equal, otherwise False. </returns>
         public static bool operator ==(string source, ResourceType target)
         {
             return target.Equals(source);
         }
 
+        /// <summary>
+        /// Compares two <see cref="ResourceType"/> objects.
+        /// </summary>
+        /// <param name="source">First <see cref="ResourceType"/> object. </param>
+        /// <param name="target"> Second <see cref="ResourceType"/> object. </param>
+        /// <returns> True if they are equal, otherwise False. </returns>
         public static bool operator ==(ResourceType source, ResourceType target)
         {
             return source.Equals(target);
         }
 
+        /// <summary>
+        /// Compares a <see cref="ResourceType"/> object with a <see cref="string"/>.
+        /// </summary>
+        /// <param name="source"> <see cref="ResourceType"/> object. </param>
+        /// <param name="target"> String. </param>
+        /// <returns> False if they are equal, otherwise True. </returns>
         public static bool operator !=(ResourceType source, string target)
         {
             return !source.Equals(target);
         }
 
+        /// <summary>
+        /// Compares a <see cref="string"/> with a <see cref="ResourceType"/> object.
+        /// </summary>
+        /// <param name="source"> String. </param>
+        /// <param name="target"> <see cref="ResourceType"/> object. </param>
+        /// <returns> False if they are equal, otherwise True. </returns>
         public static bool operator !=(string source, ResourceType target)
         {
             return !target.Equals(source);
         }
 
+        /// <summary>
+        /// Compares two <see cref="ResourceType"/> objects.
+        /// </summary>
+        /// <param name="source">First <see cref="ResourceType"/> object. </param>
```

Comment:
space.  You can use this regex in find dialog to find all instances.     .*\/\/\/.*\>[^\s\.]